### PR TITLE
Release bump to v1.1.0

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

Bump mscalendar plugin to v1.1.0

#### Improvements

b8aff5a Implement autorespond feature (#188)
596cad1 Add Autocomplete icon (#209)
19d0766 Fix autocomplete terms and help text to match available commands (#200)
5798bf3 Add Fallback to Slack Attachments (#183)
b952d3c [GH-125] Remember user last status on status change (#140)
3a6d0e7 Improve message on login flow for receiving notifications during meetings (#158)
61581f3 Use limit 20 for calendar events API call (#178)
5c6f67e Add Telemetry to MSCalendar (#152)
c0d915f Order daily summary view (#177)
3e459f7 Update license to Source Available license (#182)
e05112b Drop "E20 only" from plugin name (#208)

#### Fixes

a488054 Fix panic when Daily Summary is nil for a user (#199)
50beb11 When trying to renew an expired subscription, create new subscription instead (#131)
26fe3c5 use api for license and fix tests (#195)
b8de849 Update system admin check in api.go (#197)
62dad6e Update how to fetch diagnostic id to avoid panics (#187)

#### Maintenance

8437af7 Update README.md (#190)
45c98fd Update plugin.json (#207)
5e9b8e3 Bump lodash from 4.17.11 to 4.17.20 in /webapp (#185)
d10dbd4 Bump elliptic from 6.5.0 to 6.5.3 in /webapp (#186)
47245b0 Bump ini from 1.3.5 to 1.3.8 in /webapp (#201)
4a9daba Issue-184/integrate-golanglint-only (#196)
c751170 Add release_notes_url (#193)
aee00cc go mod tidy (#212)
5cb1e2a Bump elliptic from 6.5.3 to 6.5.4 in /webapp (#211) (showing up twice here)

### CODE REVIEW

**Review by levb — APPROVED** [review]

**Review by larkox — APPROVED** [review]

File: plugin.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "support_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/issues",
     "release_notes_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/releases/tag/v1.0.0",
```
Comment (hanzei) [inline plugin.json:7]: This should be updated

**Review by hanzei — CHANGES_REQUESTED** [review]

**Review by hanzei — APPROVED** [review]

**Review by DHaussermann — COMMENTED** [review]
Tested and passed
- Ran through release tests
  - Status changes happen as expected in appreciate scenarios
  - All settings on the config panel work as expected
  - Subscriptions, Event reminders and Status change notifications work as expxected 
  - Summary report functionality is working as expected
- Confirmed new license SKUs work as expected 
- Confirmed version bumped to `v1.1.0`
  
One last question on this... Seems the build hash no longer shows in the info command. 
image
@hanzei  I believe this is related to build tooling. Do you happen to know if this should still be showing here?

**Comment by mickmister** [comment]
@hanzei Does @DHaussermann's comment above need to be resolved before this is merged?

**Comment by DHaussermann** [comment]
@mickmister one more question here - there are 2 error that occur when running the full `make`. These errors also occur on `master` branch so, it's more a question if they must be solved before release. 

```
go install github.com/golang/mock/mockgen
../../../../pkg/mod/github.com/golang/mock@v1.4.4/mockgen/parse.go:35:2: missing go.sum entry for module providing package golang.org/x/tools/go/packages (imported by github.com/golang/mock/mockgen); to add:
	go get github.com/golang/mock/mockgen@v1.4.4
make: *** [mock] Error 1
```
and
```
Running golangci-lint
golangci-lint run ./...
gomock_reflect_459867974/prog.go:64:3: exitAfterDefer: os.Exit clutters `defer func(){...}(...)` (gocritic)
		os.Exit(1)
		^
gomock_reflect_459867974/prog.go:1: File is not `gofmt`-ed with `-s` (gofmt)

make: *** [golangci-lint] Error 1
```

**Comment by mickmister** [comment]
@hanzei Can this be merged with the issue described above?

**Comment by hanzei** [comment]
I'm fine with cutting this release even if the build number issue isn't fixed yet.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Release testing done based on comment above
- Verified version stamp fix
- No further testing needed

**Pull request merged**